pkg/test/mocks/rbac/rbac.go

```diff
@@ -154,6 +154,12 @@ func MockRbac(c echo.Context) error {
 			{
 				Permission: "content-sources:repositories:write",
 			},
+			{
+				Permission: "content-sources:content_templates:read",
```

Comment:
do these need to match the ResourceTemplates string in the permission map exactly? if so, i think this would need to be `content_sources:templates:read` etc. if not, ignore this :)

Reply:
they do thanks, i changed them and forgot to update here, good catch